examples/colab/wav2vec2_saved_model_finetuning.ipynb

```diff
@@ -0,0 +1,2441 @@
+{
```

Comment:
<p>It's also worth mentioning that this code is loading the whole dataset into memory, which is fine for the small dataset here, but will not work with the full-sized LibriSpeech dataset.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/hub/pull/788/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>